npm/vue/src/index.ts

```diff
@@ -14,14 +14,6 @@ const defaultOptions: (keyof MountOptions)[] = [
   'stylesheets',
 ]
 
-function checkMountModeEnabled () {
```

Comment:
Now we have CT runner, we can assume if the user is using `mount` they are in CT mode.